scripts/globals/garrison_data.lua

```diff
@@ -0,0 +1,622 @@
+-----------------------------------
+-- Garrison Data
+-----------------------------------
+require("scripts/globals/zone")
+-----------------------------------
+xi = xi or {}
+xi.garrison = xi.garrison or {}
+
+-- Name is Determined by Nation and LevelCap
+-- Names in order of xi.nation values (sandoria, bastok, windurst)
+xi.garrison.allyNames =
+{
+    [20] = { "Patrician",       "Recruit",         "Candidate"       },
+    [30] = { "Trader",          "Mariner",         "Scholar"         },
+    [40] = { "TempleKnight",    "GoldMusketeer",   "WizeWizard"      },
+    [50] = { "RoyalGuard",      "GoldMusketeer",   "Patriarch"       },
+    [75] = { "MilitaryAttache", "MilitaryAttache", "MilitaryAttache" },
+}
+
+-- Group Ids are different per cap due to min / max level requirements
+-- They all use the same pool at the moment, but we could also change families
+-- based on cap, which would change base stats
+xi.garrison.allyGroupIds =
+{
+    [20] = 1,
+    [30] = 2,
+    [40] = 3,
+    [50] = 4,
+    [75] = 5,
+}
+
+-- Look is Determined by Nation and LevelCap (Appears to be 4 for each outpost - More data needed)
+xi.garrison.allyLooks =
+{
+    [20] =
+    {
+        [xi.nation.SANDORIA] =
+        {
+            "0x01000C030010262000303A403A5008611B700000",
+            "0x01000A040010262019303A40195008611C700000",
+        },
+
+        [xi.nation.BASTOK] =
+        {
+            "0x01000E02000066000100010001009F0000000000",
+            "0x010001017D000000150015001500B70000000000",
+            "0x01000701361005206630664066500C6129700000",
+            "0x0100020103000300000000000000DC0000000000",
+            "0x010006020F0000000F000F000F00820082000000",
+            "0x01000402740014000000000003002C0100000000",
+        },
+
+        [xi.nation.WINDURST] =
+        {
+            "0x010007070110322032300E401550AC6000700000",
+            "0x01000E0718101820183015401850B76024700000",
+        },
+    },
+    --level 30 garrison looks
+    [30] =
+    {
+        [xi.nation.SANDORIA] =
+        {
+            "0x010006030010762076303A400650736000700000",
+            "0x01000F0300101520153015401550006000700000",
+            "0x010009040010762076303A400650736000700000",
+            "0x01000E0400101520003015401550006000700000"
+        },
+
+        [xi.nation.BASTOK] =
+        {
+            -- (Missing capture)
+        },
+
+        [xi.nation.WINDURST] =
+        {
+            "0x01000B051C1073201430144014506C6000700000",
+            "0x0100010777106920693066406950B46000700000",
+            "0x010000067D107820033066406850E96000700000",
+            "0x0100020700100220023002400250006000700000", -- Naih_Arihmepp
+        },
+    },
+    --level 40 garrison looks
+    [40] =
+    {
+        [xi.nation.SANDORIA] =
+        {
+            "0x01000E04191019201930194019506B601C700000",
+            "0x01000903191019201930194019506B601C700000"
```

Comment:
Final entries should have a trailing comma too

Reply:
Fixed all of these